src/alto-ui/Form/CheckBox/CheckBox.scss

```diff
@@ -1,33 +1,41 @@
 @import '../../scss/inc';
 
 .CheckBox {
-  margin-bottom: 10px;
+  & + & {
+    //margin-top: $spacing-small;
+  }
 }
 
 .CheckBox__input {
-  @extend %visually-hidden  
+  @extend %visually-hidden;
 }
 
 .CheckBox__label {
   font-size: $font-size-medium;
-  padding-left: 1.6em;
+  // font-size: 3rem;
+  line-height: 2em;
+  height: 2em;
```

Comment:
we should use your new `$height-default` no ?

Reply:
No, cause this is `em` not `rem`